contracts/utils/RateLimitedMinter.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "./RateLimited.sol";
+
+/// @title abstract contract for putting a rate limit on how fast a contract can mint FEI
+/// @author Fei Protocol
+abstract contract RateLimitedMinter is RateLimited {
+
+    uint256 private constant MAX_FEI_LIMIT_PER_SECOND = 10_000e18; // 10000 FEI/s or ~860m FEI/day
```

Comment:
No reason to not add in methods here to have this changeable by governor.

Reply:
Will merge as is and add in another PR